instrumentation/net/http/httptrace/otelhttptrace/clienttrace.go

```diff
@@ -53,20 +60,81 @@ func parentHook(hook string) string {
 	return hookMap[hook]
 }
 
+// ClientTraceOption allows customizations to how the httptrace.Client
+// collects information.
+type ClientTraceOption func(*clientTracer)
```

Comment:
It does not follow https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#option

The option type has to be an interface. But you can make a functional option implementation: https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#functional-options

Most of the other Go Modules in this repository should already follow this pattern.

Reply:
updated